run.sh

```diff
@@ -40,7 +35,15 @@ SLEEP_IN_SECONDS=${SLEEP_IN_SECONDS:=300}
 # so that the test takes at least as long as the time we
 # keep the container alive
 WRK_OPTIONS=${WRK_OPTIONS:="-d $((SLEEP_IN_SECONDS*2))"}
-WRK_HEADER=$(node -e "require('./index.js').printHeader()")
+
+EVENT_HUB_SHARED_ACCESS_KEY_NAME=$(echo $(echo $EVENT_HUB_CONNECTION | cut -d";" -f2) | cut -d"=" -f2)
+
+EVENT_HUB_SHARED_ACCESS_KEY=$(echo $EVENT_HUB_CONNECTION | cut -d";" -f3)
+EVENT_HUB_SHARED_ACCESS_KEY=${EVENT_HUB_SHARED_ACCESS_KEY#SharedAccessKey=}
+
+# generate SAS token using remote helper code, by default token expires 24 hours from now
+wget -O /tmp/get-sas-token.sh -q https://gist.githubusercontent.com/syedhassaanahmed/bf05bbff8d9696fb978c586881dda61f/raw/42e5bbf7ef75cb3f44b82b6b95dc77303f7e23d0/get-sas-token.sh && source /tmp/get-sas-token.sh && rm -f /tmp/get-sas-token.sh
```

Comment:
I would propose that you'd add the script `get-sas-token.sh` into this repository and call it locally instead of wget'ting it. What do you think?

The script should have:

```
set -e
set -o pipefail
```

so that if the sub-commands fail, the script itself fails as well.

Reply:
Done